project_task_autocreate/models/project_task_template.py

```diff
@@ -270,7 +270,7 @@ def get_task_vals(self, record=None):
             'priority': self.priority,
             'tag_ids': [(6, 0, self.tag_ids.ids)],
             'date_deadline': self._get_deadline(record),
-            'user_id': self._get_user_id(record).id,
+            'user_id': self._get_user_id(record),
```

Comment:
@ventilooo If you look carefully at the `get_user_id`, `get_partner_id` and `get_project_id` you will see that they all return a record. Keep that consistency.